src/assets/sass/protocol/components/_split.scss

```
@@ -0,0 +1,467 @@
+
+    @media #{$mq-md} {
+        padding: 0 get-theme('h-grid-md');
+    }
+
+    @media #{$mq-xl} {
+        padding: 0 get-theme('h-grid-xl');
+    }
+}
+
+
+.mzp-c-split-body {
+    position: relative;
+
+     > :last-child {
+        margin-bottom: 0;
+
+        > :last-child {
+            margin-bottom: 0;
+        }
+    }
+}
+
+.mzp-c-split-media {
+    position: relative;
+}
+
+.mzp-c-split-media-asset {
+    display: block;
+    max-width: 100%;
+}
+
+.mzp-c-split-bg {
+    // vertical spacing should match mzp-l-content's vertical spacing
+    padding: get-theme('v-grid-xs') 0;
+
+    @media #{$mq-md} {
+        padding: get-theme('v-grid-md') 0;
+    }
+
+    @media #{$mq-xl} {
+        padding: get-theme('v-grid-xl') 0;
+    }
+}
+
+
+/* -------------------------------------------------------------------------- */
+// Mobile only rules
+
+@media all and (max-width:#{$screen-lg - 1}) {
+
+    .mzp-c-split-body + .mzp-c-split-media,
+    .mzp-c-split-media + .mzp-c-split-body {
+        margin-top: get-theme('v-grid-xs');
+    }
+
+    .mzp-c-split-media {
+        max-width: $content-sm;
+    }
+
+    .mzp-l-split-center-on-sm-md {
+        .mzp-c-split-body,
+        .mzp-c-split-media,
+        .mzp-c-split-media-asset {
+            text-align: center;
+            margin-left: auto;
+            margin-right: auto;
+        }
+    }
+
+    .mzp-l-split-hide-media-on-sm-md {
+        .mzp-c-split-body {
+            margin-top: 0;
+        }
+
+        .mzp-c-split-media {
+            display: none;
+        }
+    }
+}
+
+
+/* -------------------------------------------------------------------------- */
+// side by side layout, float based fall back
+
+@media #{$mq-lg} {
+    .mzp-c-split-body {
+        @include border-box;
+        @include bidi(((float, left, right),));
+        width: 50%;
+        padding: 0 (get-theme('h-grid-lg') / 2);
+
+        // right-third
+        .mzp-l-split-body-narrow & {
+            width: 33.33%;
+        }
+
+        .mzp-l-split-body-wide & {
+            width: 66.66%;
+        }
+
+        .mzp-l-split-reversed & {
+            @include bidi(((float, right, left),));
+        }
+    }
+
+    .mzp-c-split-media {
+        @include border-box;
+        @include bidi(((float, right, left),));
+        width: 50%;
+        padding: 0 (get-theme('h-grid-lg') / 2);
+
+        .mzp-l-split-body-narrow & {
+            width: 66.66%;
+        }
+
+        .mzp-l-split-body-wide & {
+            width: 33.33%;
+        }
+
+        .mzp-l-split-reversed & {
+            @include bidi(((float, left, right),));
+        }
+    }
+
+}
+
+@media #{$mq-xl} {
+    .mzp-c-split-body {
+        padding: 0 (get-theme('h-grid-xl') / 2);
+    }
+
+    .mzp-c-split-media {
+        padding: 0 (get-theme('h-grid-xl') / 2);
+    }
+}
+
+
+/* -------------------------------------------------------------------------- */
+// side by side layout, grid based
+
+@supports( display:grid) {
```

Nit: spacing. Not sure it matters in CSS but maybe to be safe?
```suggestion
@supports (display:grid) {
```